Remotion/Web/Development.WebTesting.IntegrationTests/Web.Development.WebTesting.IntegrationTests.csproj

```diff
@@ -173,8 +173,7 @@
     <EmbeddedResource Include="SavedTestScreenshots\DropDownListControlObjectTest\InternetExplorer.any.DropDownList0.png" />
     <EmbeddedResource Include="SavedTestScreenshots\DropDownListControlObjectTest\InternetExplorer.any.DropDownList1.png" />
     <EmbeddedResource Include="SavedTestScreenshots\DropDownMenuControlObjectTest\DropDownMenu.png" />
-    <EmbeddedResource Include="SavedTestScreenshots\DropDownMenuControlObjectTest\InternetExplorer.any.DropDownMenu0.png" />
-    <EmbeddedResource Include="SavedTestScreenshots\DropDownMenuControlObjectTest\InternetExplorer.any.DropDownMenu1.png" />
+    <EmbeddedResource Include="SavedTestScreenshots\DropDownMenuControlObjectTest\InternetExplorer.any.DropDownMenu.png" />

```

Comment:
I'm confused about 
Remotion/Web/Development.WebTesting.IntegrationTests/SavedTestScreenshots/DropDownMenuControlObjectTest/DropDownMenu.png
and
Remotion/Web/Development.WebTesting.IntegrationTests/SavedTestScreenshots/DropDownMenuControlObjectTest/Firefox.any.DropDownMenu.png

Those have been deleted but no new files added? the test failed for all browsers, so how does this work? also, if they've been deleted, where's the change in the csproj file?

Reply:
They have both been updated, not deleted. Try enabling the rich diff for those files.

Reply:
rich diff, does this exist on github review?

Reply:
![image](https://user-images.githubusercontent.com/40357550/98662298-dcdc3000-2347-11eb-9f61-fced1b7dc632.png)

Reply:
That's sooooo obvious...